.github/workflows/main.yml

```diff
@@ -73,6 +73,8 @@ jobs:
         run: |
           if [ "$RUNNER_OS" == "Linux" ]; then
               sudo apt-get update && sudo apt-get install -y capnproto libudev-dev libssl-dev
+              sudo apt install ttf-mscorefonts-installer
```

Comment:
Trying to tackle this here too: https://github.com/swift-nav/swift-toolbox/pull/1000/commits/e9c49ce232434bd87368cb0485e51e77d70508e3